tests/test_ast.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from EduNLP.Formula.ast.ast import str2ast
+
+
+'''
+Note : Some functions are not supportd in katex
```

Comment:
Note should not be written here. Suggestion: put them in Formula or relevant documents